library/pkparse.c

```diff
@@ -529,7 +530,11 @@ static int pk_get_rsapubkey( unsigned char **p,
 
     if( ( ret = mbedtls_rsa_import_raw( rsa, *p, len, NULL, 0, NULL, 0,
                                         NULL, 0, NULL, 0 ) ) != 0 )
-        return( MBEDTLS_ERR_PK_INVALID_PUBKEY );
+    {
+        if( ret != MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED )
```

Comment:
The documentation in `pk.h` needs to be updated to add `MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED` as a possible error where applicable.

Reply:
I disagree that we should mention generic error codes in our function description, however, I'm willing to add the  generic platform errors are possible statement (`MBEDTLS_ERR_PLATFORM_HW_ACCEL_FAILED` is also an option)